torch/_inductor/runtime/compile_tasks.py

```diff
@@ -1,11 +1,10 @@
-from __future__ import annotations
```

Comment:
We should be able to keep this now, right?

Reply:
Yes, but now that we are not using `compile` it in fact does not do anything. I see no reason to keep it.

Reply:
It wasn't added for the purpose of compile, so I don't see any reason to remove it?

Reply:
Fair enough. I will add it back in.